fuzz_lightyear/request.py

```diff
@@ -48,6 +56,14 @@ def id(self) -> str:
             self.operation_id,
         )
 
+    def _format_array(
```

Comment:
It's not really clear what why we're formatting here. It might be more self-documenting if we gave a little more context like `_format_array_in_path`.